tests/conftest.py

```diff
@@ -10,11 +10,14 @@
 import yaml
 import ipaddr as ipaddress
 
+from ansible.parsing.dataloader import DataLoader
+from ansible.inventory.manager import InventoryManager
 from ansible_host import AnsibleHost
 from collections import defaultdict
-from common.devices import SonicHost, Localhost, PTFHost
+from common.devices import SonicHost, Localhost, PTFHost, FanoutHost
 
 logger = logging.getLogger(__name__)
+from common.devices import SonicHost, Localhost, PTFHost
```

Comment:
Already imported in line 17. This line can be removed.

Reply:
Fixed